src/transport/SecurePairingSession.cpp

```diff
@@ -63,7 +63,7 @@ void SecurePairingSession::Clear()
     memset(&mPoint[0], 0, sizeof(mPoint));
     memset(&mWS[0][0], 0, sizeof(mWS));
     memset(&mKe[0], 0, sizeof(mKe));
-    mNextExpectedMsg = Spake2pMsgType::kSpake2pMsgError;
+    mNextExpectedMsg = Protocols::SecureChannel::MsgType::PASE_Spake2pError;
```

Comment:
I wonder whether `using namespace Protocols::SecureChannel;` might be helpful to reduce the typing.  Or `using MsgType = Protocols::SecureChannel::MsgType`?

Reply:
Followup for this one as desired.

Reply:
> using MsgType = Protocols::SecureChannel::MsgType

Now we have taken out the prefix of MsgType, Then how we differentiate "using MsgType = Protocols::Common::MsgType" and "using MsgType = Protocols::SecureChannel::MsgType" ?

Reply:
If we have to deal with both of them here, then we can't do that, of course.  In that case maybe the best thing to do is `using namespace chip::Protocols;` or `namespace SecureChannel = Protocols::SecureChannel;`.  Again, just to make things a bit shorter; it's not a big deal either way.